### PR TITLE
feat: [CDS-68131]: Git stores supporting for service overrides and configuration in Environments

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Added supporting for git stores in config files for environments(configuration) and service overrides
Screenshot 2023-04-30 at 10 43 32

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger coverage

**Review by PratyushGar — DISMISSED** [review]

**Review by PratyushGar — DISMISSED** [review]

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx [hunk 1]
```diff
@@ -175,47 +209,199 @@ function ServiceConfigFileOverride({
...
       setEditIndex(0)
       setIsEditMode(false)
+      setConfigStore('' as ConfigFileType)
     }
     return (
       <Dialog onClose={onClose} {...DIALOG_PROPS}>
         <div className={css.createConnectorWizard}>
           <ConfigFilesWizard
-            stores={[ConfigFilesMap.Harness]}
+            stores={[ConfigFilesMap.Harness, ConfigFilesMap.Github, ConfigFilesMap.Git, ConfigFilesMap.Bitbucket]}
```
Comment (chetanvekariya-harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentDetails/ServiceOverrides/ServiceConfigFileOverride/ServiceConfigFileOverride.tsx:400]: Should this be hard-coded ? Don't we need to read from map?

**Review by chetanvekariya-harness — APPROVED** [review]

**Pull request closed without merging**